### PR TITLE
Add Cost Management page to Operations section

### PR DESCRIPTION
**Pull request by avaidyanatha**
Adds a section that goes over using the Plural Console to perform cost management operations.

### CODE REVIEW

File: pages/operations/cost-management.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+## Node Optimization
+
+### Helping the Autoscaler
+
+The Plural autoscaler does a good job of scheduling new Pods and resources onto nodes that can handle the extra workload, but cannot delete Pods, as it could lead to potentially destructive scenarios. Because of this, you may end up
+with imbalanced nodes that are overworked or underworked.
+
+To address this, you'll need the Plural Console installed. Head to the `Nodes` tab in the sidebar on the left of the Console. Here, you'll see the utilization of resources for all of your nodes. You are looking for nodes that are overworked or underworked, which can be observed by seeing if cpu and/or memory is in the red (overworked) or close to zero and green (underworked).
+
+To delete these nodes, hit the red trash can symbol to the right of the problematic node and it will deprovision, letting the autoscaler schedule the Pods it was running on a different node.
```
Comment (swoodward90) [inline pages/operations/cost-management.md:25]: Did we confirm that this works for Azure @michaeljguarino ?

File: pages/operations/cost-management.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Node Sizing at Scale
+
+If you start deploying a lot of applications and resources, you may start seeing cost go up faster than expected due to the fixed cost of installing Kubernetes and its services on every node.
+
+To control this, it is recommended that you increase your node sizes to accommodate more resources and applications per node, reducing the amount of times you're paying the fixed cost for running Kubernetes.
+
+## Kubecost
+
+To monitor your cost and keep an eye out for spend, you can install [Kubecost](https://docs.plural.sh/applications/kubecost) on Plural just how you would install any other application.
```
Comment (swoodward90) [inline pages/operations/cost-management.md:39]: Do you want to link to their docs here too?

File: pages/operations/cost-management.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+title: Cost Management
+description: Controlling the cost of your cluster and applications.
+---
+
+Out of the box, Plural performs a lot of optimizations behind the scenes with our built-in autoscaler that schedules workloads efficiently across your infrastructure. However, there are some efforts you can make to monitor and optimize this cost.
```
[the remark this answers is not in the document]
Comment (michaeljguarino) [reply pages/operations/cost-management.md:6]: In this case autoscaler is the right terminology, but both technically do optimize cost in k8s

File: pages/operations/cost-management.md (hunk 1 above)
Comment (michaeljguarino) [reply pages/operations/cost-management.md:25]: it doesn't still i think, but we should get lukasz on it soon

**Review by michaeljguarino — APPROVED** [review]
lgtm

**Pull request merged**